packages/sdk/tests/handlers/utils.test.ts

```diff
@@ -5,7 +5,7 @@
 import argon2 from 'argon2';
 import { Directus } from '../../src';
 import { test } from '../utils';
-import { generateHash } from '../utils/generate-hash';
+import { generateHash } from '../utils';
```

Comment:
Why not migrate `api/src/utils/generate-hash.ts` to `shared` so we only maintain one function.

Reply:
It relies on getConfigFromEnv() and that is going to have to be mocked entirely anyway. Though a deep mock isn't the worst idea. I'll do that.

Reply:
It also is only used in tests and not in any of the actual working files.